### ISSUE TITLE
Format data dictionary into a table

### ISSUE THREAD

**Issue by dilinie-seimon** [etc5521-2020/assignment-1-numbat#7]
For direct visibility and better user experience, convert the data dictionary (which is currently a list) into a table format

**Comment by jluo0015** [comment]
Indeed, the table format could help readers to realise the dataset more directly than the list format. In addition, when we plot the figure, it also requires the dataframe or tibble format instead of the list format. Therefore, changing the format of the data dictionary to the table format will be better.
